.env

```diff
@@ -0,0 +1,5 @@
+DB_NAME=postgres
+DB_USER=postgres
+DB_PASSWORD=test1234
+DB_HOST=localhost
+DB_DIALECT=postgres
```

Comment:
you mist to add .env file to gitignore, so other user must not see your passwords